tests/publiccloud/upload_image.pm

```diff
@@ -32,7 +32,7 @@ sub run {
 
     if (my $img_id = $provider->find_img($img_name)) {
         record_info('Info', "Image $img_id already exists!");
-        return;
+        #return;
```

Comment:
Leftover?

Reply:
> Leftover?

The second commit is for testing. But I don't even need this as I'm uploading with a new name. Fixing.